endpoints/bookstore-grpc/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:jessie
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y -q ca-certificates \
+    && apt-get -y -q upgrade \
+    && apt-get install -y python-dev python-pip virtualenv \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN virtualenv /env
```

Comment:
I should've mentioned this during the last review, but use python 3, please. `virtualenv -p python3.4`. If you use the Python runtime, use `-p python3.5`.